pyroapi/dispatch.py

```diff
@@ -146,7 +146,7 @@ def register_backend(alias, new_backends):
     'pyro': 'pyro.contrib.minipyro',
 })
 register_backend('funsor', {
-    'distributions': 'funsor.distributions',
+    'distributions': 'funsor.torch.distributions',
```

Comment:
@fehiepsi is this the only change necessary?

Reply:
Thanks, @fritzo ! The change passes my tests locally.

Reply:
Great, I'll release pyro-api. Thanks for reviewing.